spec/spec_helper.rb

```diff
@@ -11,8 +11,8 @@
 
 SlackRubyBotServer::Service.logger.level = Logger::WARN
 
-if SlackRubyBotServer::Config.postgresql?
-  require 'spec/support/databases/postgresql'
-elsif SlackRubyBotServer::Config.mongo?
-  require 'spec/support/databases/mongo'
+if SlackRubyBotServer::Config.pg?
+  require 'spec/support/database_adapters/pg/*.rb'
```

Comment:
I didn't know you could do a require with `*.rb`, learning every day :) 

This isn't a big deal, but it makes the order of requires unpredictable inside that folder. I would `require .../pg.rb` and inside `pg.rb` require anything that it may need.